charts/collabora-online/0.0.1/questions.yaml

```diff
@@ -159,5 +148,157 @@ questions:
           schema:
             type: string
             default: "-o:welcome.enable=false -o:user_interface.mode=notebookbar"
+  # Storage
+  # Configure app volumes
+  - variable: appVolumeMounts
+    group: "Storage"
+    label: ""
+    schema:
+      type: dict
+      attrs:
+        # Config ------------------------
+        - variable: config
+          label: "Config Volume"
+          schema:
+            type: dict
+            $ref:
+              - "normalize/ixVolume"
+            attrs:
```

Comment:
Besides config, you might also want your data mounted somewhere ;-)

Reply:
Thanks for pointing out! In this case collabora does not have any data, even config can be done completely with env vars passed on `extra_params`. 
Added the config folder only, because the alternative to `extra_params` is editing the xml file.